// uuid/dist/cjs/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.version = exports.validate = exports.v7 = exports.v6ToV1 = exports.v6 = exports.v5 = exports.v4 = exports.v3 = exports.v1ToV6 = exports.v1 = exports.stringify = exports.parse = exports.NIL = exports.MAX = void 0;
var max_js_1 = require("uuid/dist/cjs/max.js");
Object.defineProperty(exports, "MAX", { enumerable: true, get: function () { return max_js_1.default; } });
var nil_js_1 = require("uuid/dist/cjs/nil.js");
Object.defineProperty(exports, "NIL", { enumerable: true, get: function () { return nil_js_1.default; } });
var parse_js_1 = require("uuid/dist/cjs/parse.js");
Object.defineProperty(exports, "parse", { enumerable: true, get: function () { return parse_js_1.default; } });
var stringify_js_1 = require("uuid/dist/cjs/stringify.js");
Object.defineProperty(exports, "stringify", { enumerable: true, get: function () { return stringify_js_1.default; } });
var v1_js_1 = require("uuid/dist/cjs/v1.js");
Object.defineProperty(exports, "v1", { enumerable: true, get: function () { return v1_js_1.default; } });
var v1ToV6_js_1 = require("uuid/dist/cjs/v1ToV6.js");
Object.defineProperty(exports, "v1ToV6", { enumerable: true, get: function () { return v1ToV6_js_1.default; } });
var v3_js_1 = require("uuid/dist/cjs/v3.js");
Object.defineProperty(exports, "v3", { enumerable: true, get: function () { return v3_js_1.default; } });
var v4_js_1 = require("uuid/dist/cjs/v4.js");
Object.defineProperty(exports, "v4", { enumerable: true, get: function () { return v4_js_1.default; } });
var v5_js_1 = require("uuid/dist/cjs/v5.js");
Object.defineProperty(exports, "v5", { enumerable: true, get: function () { return v5_js_1.default; } });
var v6_js_1 = require("uuid/dist/cjs/v6.js");
Object.defineProperty(exports, "v6", { enumerable: true, get: function () { return v6_js_1.default; } });
var v6ToV1_js_1 = require("uuid/dist/cjs/v6ToV1.js");
Object.defineProperty(exports, "v6ToV1", { enumerable: true, get: function () { return v6ToV1_js_1.default; } });
var v7_js_1 = require("uuid/dist/cjs/v7.js");
Object.defineProperty(exports, "v7", { enumerable: true, get: function () { return v7_js_1.default; } });
var validate_js_1 = require("uuid/dist/cjs/validate.js");
Object.defineProperty(exports, "validate", { enumerable: true, get: function () { return validate_js_1.default; } });
var version_js_1 = require("uuid/dist/cjs/version.js");
Object.defineProperty(exports, "version", { enumerable: true, get: function () { return version_js_1.default; } });
